foreman/foreman.spec

```diff
@@ -570,6 +560,7 @@ sed -i 's/:organizations_enabled: false/:organizations_enabled: true/' config/se
 export BUNDLER_EXT_NOSTRICT=1
 export BUNDLER_EXT_GROUPS="default assets"
 %{scl_rake} assets:precompile RAILS_ENV=production --trace
+%{scl_rake} webpack:compile --trace
```

Comment:
Oh well.. there's more than 1 package needed to run webpack compile, I guess these will be the first 'bundled-npm(xxxx)' packages https://github.com/theforeman/foreman/pull/3433/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R8